aws/cf_template_test.go

```diff
@@ -614,6 +619,38 @@ func TestGenerateTemplate(t *testing.T) {
 				require.NotEqual(t, cloudformation.Integer(3), tg.UnhealthyThresholdCount)
 			},
 		},
+		{
+			name: "target port http2 when https listener and configured",
```

Comment:
there was a comment by @dhohengassner  about adding a test with GRPC TG, which we miss in case you add the flag that supports GRPC in https://github.com/zalando-incubator/kube-ingress-aws-controller/pull/460/files#diff-cc5d2cdea66aa1021f86124b6b59590af4179f66c5da18e04c43ae82c0b6bfbdR284

Reply:
done